modules/ParallelText/ParallelText-metadata.json

```diff
@@ -0,0 +1,71 @@
+{
+    "@context": [
+        "http://www.w3.org/ns/csvw",
+        {
+            "@language": "en"
+        }
+    ],
+    "dc:conformsTo": "http://cldf.clld.org/v1.0/terms.rdf#ParallelText",
+    "dialect": {
+        "commentPrefix": null
+    },
+    "tables": [
+        {
+            "url": "forms.csv",
+            "dc:conformsTo": "http://cldf.clld.org/v1.0/terms.rdf#FormTable",
+            "tableSchema": {
+                "columns": [
+                    {
+                        "name": "ID",
+                        "required": true,
+                        "propertyUrl": "http://cldf.clld.org/v1.0/terms.rdf#id",
+                        "datatype": {
+                            "base": "string",
+                            "format": "[a-zA-Z0-9_\\-]+"
+                        }
+                    },
+                    {
+                        "name": "Language_ID",
+                        "required": true,
+                        "propertyUrl": "http://cldf.clld.org/v1.0/terms.rdf#languageReference",
+                        "datatype": "string"
+                    },
+                    {
+                        "name": "Parameter_ID",
+                        "required": true,
+                        "propertyUrl": "http://cldf.clld.org/v1.0/terms.rdf#parameterReference",
+                        "datatype": "string"
+                    },
```

Comment:
@cysouw What do you think are the parameters be in this scenario? Something like "verse 1 in chapter 2 of the gospel according to Luke" or similar?

Reply:
Ah sorry, just reading the description further down.

Reply:
I guess I understand somewhat better now. But now it seems as if parallel text is **very** similar to a wordlist with cognates - maybe too similar? Could we add an example, preferably one that highlights the difference?

Reply:
They are extremely similar, with two main differences:

- the alignments are not cognates
- the 'trick' of using spaces to align chunks in the string only works for highly similar languages. In most cases you need a more general approach to alignment (what I call 'explicit reference', or 'stand-off' reference if you want)

Reply:
As for the parameters: yes, I use IDs like 40002017 which breaks down to `Markus 2:17`.

Reply:
Ok, so the forms actually may be complete verses?

Reply:
In case of the bible parallel text, the forms **are** complete verses.

Reply:
I think this should be stressed in the README, because from my point of view that's the main argument for specifying this as separate module.

Reply:
I'll add some more to the readme in a minute